engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -1558,6 +1558,10 @@ public void run()
           entry.getValue().recordingId = null;
         }
         for (ContainerStats.OperatorStats stats : statsList) {
+          if (stats == null) {
```

Comment:
I'm going to merge it, but shouldn't we ensure in the container that we don't put null elements into the heartbeat data?